src/pages/game/is-guessed-toast/isGuessedToast.tsx

```diff
@@ -0,0 +1,28 @@
+import { Alert, Snackbar } from '@mui/material';
+
+import { useAppSelector } from '../../../app/hooks';
+import { selectGuessedWords } from './isGuessedToastSlice';
+
+type PropsType = {
+    autoHideDuration: number;
+    onToastClose: () => void;
+};
+
+const AlreadyGuessedToast = ({ autoHideDuration, onToastClose }: PropsType) => {
+    const { isAlreadyGuessed } = useAppSelector(selectGuessedWords);
+
+    return (
+        <Snackbar
+            open={isAlreadyGuessed}
+            autoHideDuration={autoHideDuration}
+            onClose={onToastClose}
+            anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
```

Comment:
It would look better on top, on bottom it will overlay input which is bad UX. I think it would be the best to position it right below the top bar overlaying the article, so please try to achieve that.

Reply:
Fixed.